jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetLambdaHandlerResolver.kt

```diff
@@ -100,6 +80,39 @@ class DotNetLambdaHandlerResolver : LambdaHandlerResolver {
         return isMethodExists
     }
 
+    /**
+     * Disable frontend line markers since Rider get them from backend.
+     * Please see [software.aws.toolkits.jetbrains.services.lambda.LambdaHost]
+     */
+    override fun shouldShowLineMarker(handler: String): Boolean = false
```

Comment:
Is this needed since you now registered the line marker per language instead of Any?

Reply:
Removed.